index.js

```diff
@@ -1705,7 +1705,14 @@ class GithubScm extends Scm {
         const prComments = await this.prComments(scmInfo, prNum, token);
 
         if (prComments) {
-            const botComment = prComments.comments.find(commentObj => commentObj.user.login === this.config.username);
+            const botComment = prComments.comments.find(
+                commentObj =>
+                    commentObj.user.login === this.config.username &&
+                    commentObj.body.split(/\n/)[0].match(/^.+pipelines\/(\d+)\/builds.+ ([\w-:]+)$/) &&
```

Comment:
might be easier to read if it's saved as a variable
but not blocking

Reply:
I stored the regular expression in a global variable.